Obsidian/Entities/Player.cs

```diff
@@ -880,6 +880,57 @@ private async Task PickupNearbyItemsAsync(float distance = 0.5f)
         }
     }
 
+    internal async Task UpdateChunksAsync(bool unloadAll = false)
```

Comment:
I would argue that updating chunks is Client and not Player, but I think @Tides agrees with you too so majority rules.

Reply:
fun fact: I personally agree on chunks being a client thing.